cmd/influxd/inspect/report_tsi1.go

```diff
@@ -0,0 +1,98 @@
+package inspect
+
+import (
+	"errors"
+	"io"
+	"os"
+	"path"
+
+	"github.com/influxdata/influxdb"
+
+	"github.com/influxdata/influxdb/logger"
+	"github.com/influxdata/influxdb/tsdb/tsi1"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap/zapcore"
+)
+
+// Command represents the program execution for "influxd inspect report-tsi".
+var reportTSIFlags = struct {
+	// Standard input/output, overridden for testing.
+	Stderr io.Writer
+	Stdout io.Writer
+
+	// Data path options
+	Path           string // optional. Defaults to dbPath/engine/index
+	SeriesFilePath string // optional. Defaults to dbPath/_series
+
+	// Tenant filtering options
+	Org    string
+	Bucket string
+
+	// Reporting options
+	TopN          int
+	ByMeasurement bool
+	byTagKey      bool // currently unused
+}{}
+
+// NewReportTsiCommand returns a new instance of Command with default setting applied.
+func NewReportTSICommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "report-tsi",
+		Short: "Reports the cardinality of tsi files short",
+		Long:  `Reports the cardinality of tsi files long.`,
```

Comment:
I'd like to see some real documentation here. For a guide on detail and verbosity take a look at this one: 

https://github.com/influxdata/influxdb/blob/bbc37b4c6814d428d2e423fbac777a9139640be5/cmd/influxd/inspect/report_tsm.go#L29-L57